src/Dev/Tasks/TagsToShortcodeHelper.php

```diff
@@ -211,7 +211,7 @@ private function getTagsInContent($content)
         $resultTags = [];
 
         $regex = '/<('.$this->validTagsPattern.')\s[^>]*?('.$this->validAttributesPattern.')\s*=.*?>/i';
-        preg_match_all($regex, $content, $matches, PREG_SET_ORDER);
+        preg_match_all($regex ?? '', $content ?? '', $matches, PREG_SET_ORDER);
```

Comment:
```suggestion
        preg_match_all($regex ?? '//', $content ?? '', $matches, PREG_SET_ORDER);
```

Reply:
Though in this case we can just trust that it's a valid regex based on how it's assigned above.